doc/source/conf.py

```diff
@@ -20,15 +20,22 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
+from pkg_resources import get_distribution
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+release = get_distribution('satpy').version
```

Comment:
Copy&paste error: `satpy` -> `trollflow2`

Reply:
```suggestion
release = get_distribution('trollflow2').version
```

Reply:
@adybbroe could you commit the suggestion above, so we can merge this ?